hepcrawl/spiders/desy_spider.py

```diff
@@ -287,7 +287,7 @@ def _create_json_record(xml_record):
                 try:
                     hep_record = marcxml2record(xml_record)
                 except Exception as e:
-                    return {'xml_record': xml_record, 'error': e,
+                    return {'xml_record': xml_record, 'error': str(e),
```

Comment:
wouldn't it better to use `repr` instead of `str` here?
```
In [1]: e = ValueError('foo')

In [2]: print(str(e), repr(e))
foo ValueError('foo',)
```

Reply:
I agree :)

Reply:
Though this needs a big refactor anyhow to actually use the protocol thingie and show a proper message and such, so for me it can go as is now, no biggie.